src/components/charts/BarChart.tsx

```diff
@@ -114,15 +120,16 @@ const BarChart = () => {
   }, [xAxisLabels, azureSecondaryData]);
 
   const getSessionData = async () => {
-    // const params = {
-    //   period: duration,
-    //   starttime: startDateTime,
-    //   channel: channel,
-    // };
+    const params = {
```

Comment:
I commented this out because I got typescript error for unused variable.
how come you didn't get the same error ?

Reply:
ran "npm run build" which does not give any error. Can you accept this for now since 'params' must be used once API implementation takes place?